hash/hasher_bcrypt.go

```diff
@@ -24,15 +28,25 @@ func NewHasherBcrypt(c BcryptConfiguration) *Bcrypt {
 
 func (h *Bcrypt) Generate(ctx context.Context, password []byte) ([]byte, error) {
 	if err := validateBcryptPasswordLength(password); err != nil {
-		return nil, err
+		return nil, errors.WithStack(err)
 	}
 
 	hash, err := bcrypt.GenerateFromPassword(password, int(h.c.Config(ctx).HasherBcrypt().Cost))
 	if err != nil {
-		return nil, err
+		return nil, errors.WithStack(err)
+	}
+
+	var b bytes.Buffer
+	if _, err := fmt.Fprintf(
+		&b,
+		"$%s%s",
```

Comment:
Let's stay consistent?

```suggestion
		"$%s$%s",
```

I mean, `bcrypt` is a prefix of `bcryptAes`, so we should really have the delimiter here.

Reply:
OK. That will need a bit of work. Will do that in separate commit.

Reply:
Actually I remember now that we don't want to add this `$` here since BCrypt hashes have it already i.e. `$2a$15$GRvRO2nrpYTEuPQX6AieaOlZ4.7nMGsXpt.QWMev1zrP86JNspZbO` so it will be consistent with other functions.